tests/neg/i10810.scala

```diff
@@ -0,0 +1,11 @@
+final case class A()
+final case class B(a:A)
+
+object Test:
```

Comment:
This file and commit reference the wrong issue, #10810 is for command line abbreviations

Reply:
Right. It was #10870.